components/calendar/calendar.js

```diff
@@ -764,20 +765,18 @@ class Calendar extends LocalizeCoreElement(RtlMixin(LitElement)) {
 		const oldFocusDate = new Date(this._focusDate);
 		if (!isDateInRange(possibleFocusDate, getDateFromISODate(this.minValue), getDateFromISODate(this.maxValue))) {
 			// if date is not in range but we are in a dialog, _focusDate should become min or max date if possible
-			if (this._dialog) {
```

Comment:
Nothing changed here - I just was initially in here looking for the problem and cleaned it up a bit while here.